### PR TITLE
Update addresses types according to setting

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: packages/app/src/setting/scripts.ts [hunk 1]
```diff
@@ -27,20 +27,17 @@ const mainnetScripts = new Map<string, LockScript>([
...
         index: '0x0'
       },
       depType: 'depGroup'
     }])
-
-
   ],
   [
-    'any one can pay',
+    'anyone-can-pay',
```
Comment (Keith-CY) [inline packages/app/src/setting/scripts.ts:40]: Why add `-` in the script name?

Comment (duanyytop) [reply packages/app/src/setting/scripts.ts:40]: This is a more standard way of writing. https://talk.nervos.org/t/rfc-anyone-can-pay-lock/4438

Comment (Keith-CY) [reply packages/app/src/setting/scripts.ts:40]: I see, but we are going to use https://github.com/nervosnetwork/keyper/blob/d324671d2dc6e886e0a7a5cc102d7c3a3ed62335/packages/container/src/locks/anyone-can-pay/index.ts#L8 as the name of the script 😂

Comment (duanyytop) [reply packages/app/src/setting/scripts.ts:40]: Maybe we should discuss the standard of script name for other custom scripts

Comment (Keith-CY) [reply packages/app/src/setting/scripts.ts:40]: > Maybe we should discuss the standard of script name for other custom scripts

User imported scripts will show the `script.name` property, and I'm fixing the system scripts now.

**Pull request merged**